releasing/README.md

```diff
@@ -110,196 +110,72 @@ infrequently).
 Executables appear on the [release page].
 The tag appears in the URL, e.g. [pluginator/v1.0.0].
 
-## Release procedure
+## NEW Release procedure
 
-> TODO: script what follows, so someone can enter
-> ```
-> go run ./releasing/release.go kustomize minor
-> # or:
-> # go run ./releasing/release.go api patch
-> # go run ./releasing/release.go pluginator minor
-> ```
-> The program would look at the existing remote tags,
-> confirm sanity and increment the appropriate major/minor/patch
-> component, create the right branch and tag, etc.
-> No more bash please.
+For old instructions, see [legacyRelease.md](legacyRelease.md)
 
-At any given moment, the repository's master branch is
-passing all its tests and contains code one could release.
+### Update releasing/VERSIONS
 
-### get up to date
+Update the releasing/VERSIONS file with the new release
+version for the modules being released.
 
-```
-git fetch upstream
-git checkout master
-git rebase upstream/master
-```
-
-### select a module to release
-
-```
-module=pluginator  # The pluginator executable
-module=kustomize   # The kustomize executable
-module=api         # The API
-```
-
-### review tags to help determine new tag
-
-Local:
-```
-git tag -l | grep $module
-```
-
-Remote:
-```
-git ls-remote --tags upstream | grep $module
-```
-
-### determine the version
-
-Go's [semver]-compatible version tags take the form `v{major}.{minor}.{patch}`:
-
-| major | minor | patch |
-| :---:   | :---:  | :---: |
-| API change | enhancements | bug fixes |
-| manual update | maybe auto-update | auto-update encouraged |
-
- - If there are only bug fixes or refactors, increment `patch` from whatever it is now.
- - If there are new features, increment  `minor`.
- - If there's an API change (either the Go API or the CLI behavior
-   with respect to CLI arguments and flags), increment `major`.
-
-```
-major=0; minor=1; patch=0
-```
-
-### create the release branch
-
-Every module release occurs on it's own git branch.
-
-The branch name doesn't include the patch number,
-since the branch accumulates patch releases.
-
-> TODO: define procedure for doing a cherrypick (committing a patch) to a
-> release branch that already exists.
-
-Name the branch:
-
-```
-branch="release-${module}-v${major}.${minor}"
-echo "branch=$branch"
-```
-
-Create it:
-```
-git checkout -b $branch
-```
-
-### define the release tag
+For modules with a major version of 1 or greater:
+ 
+- if no new features are added in the release and it is only bug fixes
+  - update the patch version: `kustomize_patch=0` => `kustomize_patch=1`
+- if new features are added in the release
+  - update the minor version: `kustomize_minor=5` => `kustomize_minor=6`
+  - set the patch version to 0 
+- if the API surface is changed in a non-backward compatible way
+  - update the major version `kustomize_major=3` => `kustomize_major=4`
+  - set the minor version to 0
+  - set the patch version to 0
```

Comment:
FWIW, all i want is
```
release (kustomize|api|kyaml|...) (major|minor|patch)
```
per the TODO being deleted.

The script doing this - which i think should be a go program for sanity - can get the right digits by consulting existing git tags.

There's enough information in the tags to get this done without error.

BUT am willing to live with this as a stopgap